pkg/tplink_adapter.py

```diff
@@ -25,8 +25,39 @@ def __init__(self, verbose=False):
                          verbose=verbose)
 
         self.pairing = False
+        self.add_from_config()
         self.start_pairing(_TIMEOUT)
 
+    def add_from_config(self):
+        """Attempt to add all configured devices."""
+        database = Database('tplink-adapter')
+        if not database.open():
+            return
+
+        config = database.load_config()
+        database.close()
+
+        if not config or 'addresses' not in config:
+            return
+
+        for address in config['addresses']:
+            try:
+                dev = Discover.discover_single(address)
```

Comment:
Does an error of some type get logged if the discover fails?

Reply:
It does now.